### ISSUE TITLE
"Upload a File" Button Breaks Read Order

### ISSUE THREAD

**Issue by Santoz2** [ManifoldScholar/manifold#2438]
In all instances of the Upload a File button, the text inside the button is broken up in an odd way. "Upload a File" is hidden, so the rest of the text following it is read first.

## Expected Behavior
All content that is visible on a page should be read to a non-visual user. 

## Current Behavior
"Upload a File" is given an "aria-hidden='true'" attribute, and is thereby not read. Because of this the sentence it was a part of becomes a bit confusing.

## Steps to Reproduce
1. Navigate to Admin Interface (via Enter Admin Mode)
2. Navigate to an editable project, landing on its General Page
3. Navigate to the Project Thumbnail Section
4. Use a Screen Reader's Browse Mode (NVDA Modifier + Space with NVDA)
5. Use Down Arrow multiple times to read through the contents of the Custom Thumbnail button
6. Note how it is read out as "Clickable Or - Drag and drop here - GIF, JPEG, JPG, PNG - Clickable Button Browse". 
7. Adding to the confusion, this particular button does not have a focus indicator, but I'll make that a separate issue.

## Testing Environment
NVDA 2019.2.1
ESR Firefox 68.2.0 esr

**Comment by Santoz2** [comment]
The lack of focus indicator is noted in Issue #2439

**Comment by Santoz2** [comment]
4/27/2020 UW ATX: Verifed as fixed; reading this widget now announces a proper read order: "clickable BACKGROUND IMAGE Clickable button Browse... ... UPLOAD A FILE OR DRAG AND DROP HERE GIF, JPEG, JPG, PNG."
